### PR TITLE
Corrected some grammar and spelling errors.

### PR DESCRIPTION
**Pull request by jjm223**
I've noticed a few errors here and there regarding text in game.

### CODE REVIEW

File: MyPet_en.properties [hunk 1]
```diff
@@ -15,51 +15,51 @@ Message.CaptureHelper.NotUserCreated=This Iron Golem <red>wasn't<reset> build by
...
-Message.Command.Help.Type=Shows info about MyPet-Types
-Message.Command.Name.New=The name of your pet is now: <aqua>{0}<r>
+Message.Command.Help.Stop=Stops your pet from attacking  (alias: /ps or /pets)
+Message.Command.Help.Type=Shows info about MyPet types
+Message.Command.Name.New=Your pet's name is now: <aqua>{0}<r>
 Message.Command.Name.ToLong="<r>{0}<r>" is too long. The maximum lenght is <gold>{1}<r> characters.
-Message.Command.PetType.Unknown=<red> Unknown Pettype!
-Message.Command.Release.Dead=<aqua>{0}<reset> is <red>dead<reset> and will never come back. . .
+Message.Command.PetType.Unknown=<red>Unknown pet type!
+Message.Command.Release.Dead=<aqua>{0}<reset> has been released, and is no longer alive...
```
Comment (xXKeyleXx) [inline MyPet_en.properties]: Even if the old one wasn't correct (IDK) it sounded way better. The released part may need some change.

File: MyPet_en.properties [hunk 2]
```diff
@@ -15,51 +15,51 @@ Message.CaptureHelper.NotUserCreated=This Iron Golem <red>wasn't<reset> build by
...
 Message.Hunger.Hungry=<aqua>{0}<reset> is very hungry. You should feed <aqua>{0}<reset> <green>now<reset>.
 Message.Hunger.Rumbling=<aqua>{0}<reset>'s stomach is rumbling. You should feed <aqua>{0}<reset> <gold>soon<reset>.
 Message.Hunger.Starving=<aqua>{0}<reset> is starving. Feed <aqua>{0}<reset> <red>NOW<reset>!
-Message.Leash.Add=<green>You took a pet on a lead, it will be a good MyPet.
-Message.LevelSystem.LevelUp=<aqua>{0}<reset> is now Lv<gold>{1}<reset>
+Message.Leash.Add=<green>You've tamed a pet! It will be a good MyPet.
+Message.LevelSystem.LevelUp=<aqua>{0}<reset> is now level <gold>{1}<reset>.
 Message.LevelSystem.ReachedMaxLevel=<aqua>{0}<reset> has achieved the level cap of <gold>{1}<reset>.
-Message.MultiWorld.NoActivePetInThisWorld=You <red>don't<reset> have a <gold>MyPet<reset> in this world.
+Message.MultiWorld.NoActivePetInThisWorld=You don't have a <gold>MyPet<reset> in this world!
```
Comment (xXKeyleXx) [inline MyPet_en.properties]: Why no red color?

**Comment by xXKeyleXx** [comment]
I can't belive that after all these years someone corrected it xD Thanks alot.

**Comment by jjm223** [comment]
Regarding your notes, 
I feel that "is dead and will never come back" doesn't really sound correct, as this is referring to when an animal is released and disappears, not just to when it dies. I agree that my replacement still sounds a little odd though.

I started removing the red color because in my mind it seems to be a bit over the top, and it doesn't really serve much of a purpose. After I was finished I had started to put the red back as I realized it wouldn't be consistent with the other translations, I guess I forgot a line :P

I fixed these two issues. I modified my "released forever" replacement to sound a bit better. I feel that it's a bit more fitting than the original message.

**Comment by xXKeyleXx** [comment]
"This branch has conflicts that must be resolved" Could you please do a quick rebase so I can merge it from here?

**Comment by jjm223** [comment]
I took care of it, in case you didn't see.

**Comment by xXKeyleXx** [comment]
Thanks :D

**Pull request merged**